parsl/launchers/launchers.py

```diff
@@ -365,6 +365,51 @@ def __call__(self, command, tasks_per_node, nodes_per_block, walltime=None):
 aprun -n {tasks_per_block} -N {tasks_per_node} {overrides} /bin/bash cmd_$JOBNAME.sh &
 wait
 
+echo "Done"
+'''.format(command, tasks_per_block,
+           tasks_per_block=tasks_per_block,
+           tasks_per_node=tasks_per_node,
+           overrides=self.overrides)
+        return x
+
+class JsrunLauncher(Launcher):
+    """  Worker launcher that wraps the user's command with the Jsrun launch framework
+    to launch multiple cmd invocations in parallel on a single job allocation
+
+    """
+    def __init__(self, overrides=''):
+        """
+        Parameters
+        ----------
+
+        overrides: str
+             This string will be passed to the aprun launcher. Default: ''
+        """
+        self.overrides = overrides
+
+    def __call__(self, command, tasks_per_node, nodes_per_block, walltime=None):
+        """
+        Args:
+        - command (string): The command string to be launched
+        - tasks_per_node (int) : Workers to launch per node
+        - nodes_per_block (int) : Number of nodes in a block
+
+        KWargs:
+        - walltime (int) : This is not used by this launcher.
```

Comment:
don't implement fake arguments

Reply:
although that's probably a problem in the superclass interface, not here.

Reply:
I think we should update the launcher interface definition if we were to remove the walltime kwarg for consistency. I don't think we are using the walltime option anyway, so removing it "should" be safe. What do you think ? I've removed the docstring.

Reply:
I agree